src/components/Button/config.ts

```diff
@@ -0,0 +1,49 @@
+import { colorShades, flatColors } from '../../theme/palette';
+
+type ButtonConfig = {
+  sizes: {
+    [key in 'lg' | 'md' | 'sm' | 'default']: number;
```

Comment:
this and fontSize could be done like flatColors typing using `as const`

Reply:
Also moved it to button base. Don't know why I created it on Button